utils/error-reporter.ts

```diff
@@ -1,29 +1,14 @@
 import axios from "axios";
 import { ConsolaLogObject } from "consola";
-import uuid from "uuid-random";
+import { v4 as uuidv4 } from 'uuid'
 
 // We don't use logger here, because logger uses this module
 
+//TODO: make it configurable?
```

Comment:
This is a great idea :)

Reply:
For now I've added mode.isProd check (so errors will be sent only on prod env.)